doc/modules/ROOT/pages/end-to-end-examples/load-data-via-graph-construction.adoc

```diff
@@ -0,0 +1,167 @@
+= Load data to a projected graph via graph construction
+
+[TIP]
+====
+Jupyter Notebook available on https://github.com/neo4j/graph-data-science-client/blob/main/examples/load-data-via-graph-construction.ipynb[Github^]
```

Comment:
This link should probably point to the branch that corresponds to the pip installed version below

Reply:
I guess we can use `{docs-version}` like we do in https://github.com/neo4j/graph-data-science-client/blob/main/doc/modules/ROOT/pages/index.adoc for this :) (below as well)

Reply:
Yes, makes sense. Fixed.